packages/libs/react-ui/src/components/Box/Box.stories.tsx

```diff
@@ -12,49 +12,126 @@ const selectOptions: (keyof typeof vars.sizes | undefined)[] = [
 
 const meta: Meta<IBoxProps> = {
   title: 'Layout/Box',
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'Box is the most basic building block of application layout.\n' +
+          '\nThis component allows for passing the <i>margin</i> and <i>padding</i> properties.',
```

Comment:
I would also add `display`

Reply:
@eileenmguo do I add the `display` property to the component as well?